hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerStateMachine.java

```diff
@@ -198,8 +210,12 @@ public TransactionContext startTransaction(
       // only after completing 101 - 149. In initial stage, we are starting
       // with single global executor. Will revisit this when needed.
 
+      // Add the term index and transaction log index to applyTransaction map
+      // . This map will be used to update lastAppliedIndex.
+      applyTransactionMap.put(trxLogIndex, trx.getLogEntry().getTerm());
```

Comment:
Do we need to put all log indices in the applyTransactionMap? While updating lastAppliedIndex, we would want to compare with the RaftConfChange and MetadataChange entries only right?  I might be missing something here but we could optimize by adding only the required entries?

Reply:
Yes, this is done so that in the future when we have multiple executors per volume, this will help.